tests/wpunit/ContentNodeInterfaceTest.php

```diff
@@ -351,5 +351,43 @@ public function testContentNodeFieldByQueryArgUri() {
 		$this->assertEquals( $page_id, $actual['data']['page']['pageId'] );
 	}
 
+	/**
+	 * @throws Exception
+	 */
+	public function testContentNodeFieldBySlug() {
+
+		$args = array(
+			'public'              => false,
+			'show_in_graphql'     => true,
```

Comment:
@ChrisWiegman let's maybe add 2 variations of this test. 

one where the post type is: 

`'public' => false`

And another where the post_type is: 

```php
...
'public' => false,
'publicly_queryable' => true,
```

If the post_type is set to false, when #2135 is merged, the "correct" behavior will be to not return a post of a `public=>false` post_type to a public user, but if `public=>false, publicly_queryable=>true` then the posts of that post_type can be returned to public users in GraphQL.

Reply:
I've added cases with the requested configs. For now they don't meet your description above (false to both still returns the post) but I think that's intended until another PR is merged?

Reply:
Correct. I'll get both PRs merged and ensure tests are accurate for the new behavior.